pkgs/misc/gridsite/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib, stdenv, fetchFromGitHub
+, perl, doxygen, plantuml, libtool, gsoap, pkg-config
+, canl-c, libkrb5, curl, apr, apacheHttpd, aprutil, zlib
+, openssl, libxml2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gridsite";
+  version = "unstable-2018-12-13";
+
+  src = fetchFromGitHub {
+    owner = "CESNET";
+    repo = "gridsite";
+    rev = "12d6a775570cdb6b405f0f7d40784819a531cc4f";
+    sha256 = "sha256-1oAW/rTXynnogvG2ttiBJ1rpgK2t527//qtAMCK27/s";
+  };
+
+  postPatch = ''
+    patchShebangs src/roffit
+  '';
+
+  nativeBuildInputs = [ perl doxygen plantuml libtool gsoap pkg-config ];
+  buildInputs = [ canl-c libkrb5 curl apr apacheHttpd aprutil gsoap zlib ];
+  propagatedBuildInputs = [ openssl libxml2 ];
+
+  preBuild = ''
+    cd src
+
+    # gsoap is built statically in nixpkgs so --static needs to be included in the pkg-config call
+    makeFlagsArray+=(
+      "GSOAPSSL_LIBS=$(pkg-config gsoapssl --libs --static)"
+    )
+  '';
+
+  makeFlags = [
+    "prefix=$(out)"
+    "GSOAPDIR=${gsoap}"
+  ];
+
+  postInstall = ''
+    moveToOutput sbin "$cgi"
+    moveToOutput lib "$lib"
+  '';
+
+  outputs = [ "out" "cgi" "lib" "dev" "doc" "man" ];
+
+  meta = with lib; {
+    description = "Grid Security for the Web";
+    longDescription = ''
+      GridSite was originally a web application developed for managing and formatting the content of http://www.gridpp.ac.uk.
```

Comment:
80-char limit, please.

Reply:
We do not have a policy for 80 char limits afaik. Please avoid using the review process for personal preferences like this.

Reply:
`longDescription is completely optional, however you are implementing it nonetheless. 
"Personal preferences" is not a good reason to be passive-agressive, isn't it?

On the other hand it is clearly visible that your personal preference of very long lines unnecessarily complicates the reading. Indeed, GitHub web interface shows a horizontal bar on your code because of your "personal preference".

Therefore, my suggestion is not baseless.

But frak it, do as you wish.`